src/stores/useAccountStore/useAccountsStore.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { act, renderHook } from '@testing-library/react';
+
+import { useAccountStore } from './useAccountStore';
+
+describe('useAccountStore', () => {
+  it('should update mainContent correctly', () => {
```

Comment:
eu recomendaria não utilizar should pq dá a ideia de uma sugestão no teste, e o teste não é pra ser sugestivo, e sim oq deve acontecer
eu colocaria
```suggestion
  it('updates mainContent', () => {
```
mas é uma sugestão!

Reply:
Outro motivo de rancar o should é a repetição, quando aplicamos o should todo teste inicia com "should" oque é um boilerplate desnecessário uma vez que a função `it` nasceu justamente pra melhorar a leitura, antigamente era tudo `test`

coisa que antes era `test('it renders...')` poderia ser `it('renders....'` evitando repetição na descrição do teste